### ISSUE TITLE
Error: Another hypercore is stored here

### ISSUE THREAD

**Issue by noffle** [cabal-club/cabal-cli#112]
I've seen this a few times when starting up `cabal`.

```
Error: Another hypercore is stored here
    at onopen (/home/sww/.npm-global/lib/node_modules/cabal/node_modules/hypercore/index.js:294:17)
    at done (/home/sww/.npm-global/lib/node_modules/cabal/node_modules/hypercore/lib/storage.js:280:10)
    at /home/sww/.npm-global/lib/node_modules/cabal/node_modules/hypercore/lib/storage.js:255:9
    at Request.loop [as _callback] (/home/sww/.npm-global/lib/node_modules/cabal/node_modules/hypercore/lib/storage.js:356:25)
    at Request.callback (/home/sww/.npm-global/lib/node_modules/cabal/node_modules/random-access-storage/index.js:161:8)
    at onread (/home/sww/.npm-global/lib/node_modules/cabal/node_modules/random-access-file/index.js:81:31)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:658:17)
```

**Comment by noffle** [comment]
Fixed in multifeed@4.0.1 by https://github.com/kappa-db/multifeed/commit/bac39c1bba73ba9c5f4db226d62df58bdf8a29f5

**Comment by noffle** [comment]
To be fixed, this still requires `cabal-core` to be upgraded to use `kappa-core@4.x.x`. We're still using multifeed@3.x.x.

**Comment by cblgh** [comment]
seems to be fixed as of `cabal-core@6.0.0`? closing
